test/core/tests/card_timeout.py

```diff
@@ -10,7 +10,9 @@ class CardTimeoutTest(MetaflowTest):
 
     PRIORITY = 2
 
-    @tag('card(type="test_timeout_card",timeout=10,options=dict(timeout=20),save_errors=False)')
+    @tag(
+        'card(type="test_timeout_card",timeout=10,options=dict(timeout=20),save_errors=False)'
```

Comment:
yea, as @savingoyal noted, #885 tells `black` to ignore this line, because otherwise something about `test_flow.py` codegen gets messed up when this is made multiline